src/app/shared/mocks/search.mock.ts

```diff
@@ -0,0 +1,32 @@
+export class MockSearchApi {
```

Comment:
Since this is a `MockSearchApi`,
- Give it a type for `type checking` you might be testing later.
- Format should be

`export class MockSearchApi: SearchApi = {
...
}